packages/web/src/scss/components/Dropdown/README.md

```diff
@@ -221,6 +221,19 @@ modifiers in the camelCase format: `Dropdown--top`, `Dropdown--rightTop`, `Dropd
 Current class combinations (`Dropdown--top.Dropdown--left`, `.Dropdown--top.Dropdown--right`,
 `.Dropdown--bottom.Dropdown--left`, `.Dropdown--bottom.Dropdown--right`) will be removed in the next major release.
 
+## Feature Flag: Enhanced Shadow
+
+The enhanced shadow feature is disabled by default. To enable it, you have two options:
+
+1. To enable it globally, set the `$enable-dropdown-enhanced-shadow` feature flag to `true`. See main
+   [README][readme-feature-flags] for more info.
+2. Another option is to use the `spirit-enable-dropdown-enhanced-shadow` CSS class on any parent of the
+   dropdown component. Eg. on `body`.
+
+### ⚠️ DEPRECATION NOTICE
```

Comment:
Link to the deprecation section in the main readme?

Reply:
Do you mean this section? I'm not sure how much help it would provide… 🤔 

```md
## Deprecations

This package uses the deprecation warnings for props, functions and components that will be removed or replaced in the next major release.
Check your browser console to see if you are using any of the deprecated functionality.

![Deprecations in the Browser's console](https://github.com/lmc-eu/spirit-design-system/blob/main/static/deprecations-browser-console.png?raw=true)
```

But if we believe it's useful, there could be a link after the following paragraph, like this:

```md
### ⚠️ DEPRECATION NOTICE

The enhanced shadow will replace current shadow in the next major release.

[What are deprecations?][…]